build-npm-module/index190.js

```diff
@@ -0,0 +1,21 @@
+const getUpperCase = (str) => str.toUpperCase();
+
+const getLowerCase = (str) => str.toLowerCase();
+
+const getSentenceCase = (str) => {
+  const lowerStr = getLowerCase(str);
+
+  return lowerStr.charAt(0).toUpperCase() + lowerStr.slice(1)
+};
+
+const getProperCase = (str) => str.split(' ').map((word) => getSentenceCase(word)).join(' ');
+
+module.exports = {
+  getUpperCase,
+  getLowerCase,
+  getSentenceCase,
+  getProperCase
+};
+
+// Now you have finished your case-coverter module. Let's publish it to NPM.
+// You should first go to npmjs.com to get an account.
```

Comment:
We want this to work offline so we don't want to require external websites. Some things may need to be mocked. Maybe this could say something like:

Normally you would need to create your own account on npmjs.com. But for now, just use this demo account: username: camper, password: camper